src/features/lightspeed/lightspeedUser.ts

```diff
@@ -87,13 +87,16 @@ export class LightspeedUser {
   }
 
   private logAuthProviderDebugHints() {
+    const provider = this._settingsManager.settings.lightSpeedService.provider;
     const lightspeedUri = getBaseUri(this._settingsManager);
     this._logger.info(
-      `[ansible-lightspeed-user] Initializing LightspeedUser for Lightspeed URI ${lightspeedUri} and extension host ${this._extensionHost}`,
-    );
-    this._logger.info(
-      `[ansible-lightspeed-user] Red Hat authentication extension is${vscode.extensions.getExtension("redhat.vscode-redhat-account") ? "" : " not"} installed.`,
+      `[ansible-lightspeed-user] Initializing LightspeedUser with provider: ${provider}, URI: ${lightspeedUri || "(none)"}, extension host: ${this._extensionHost}`,
     );
+    if (provider === "wca") {
```

Comment:
This if condition log is not required as lighspeed auth login doesn't rely on Red Hat authentication extension.